runtime/src/chpl-privatization.c

```diff
@@ -71,8 +72,22 @@ void chpl_newPrivatizedClass(void* v, int64_t pid) {
 }
 
 
-extern void* chpl_getPrivatizedClass(int64_t i) {
+void* chpl_getPrivatizedClass(int64_t i) {
   return chpl_privateObjects[i];
 }
 
 
+void chpl_clearPrivatizedClass(int64_t i) {
+  chpl_privateObjects[i] = NULL;
+}
+
+// Used for to check for leaks of privatized classes
+int64_t chpl_numPrivatizedClasses(void) {
```

Comment:
Yes, but the locking concerns are the same -- we'd probably need to keep such a counter variable under the privatizationSync lock. I think that means it would have similar overhead, except for the obvious slowness in chpl_numPrivatizedClasses. But I wrote that as a debugging method - so I'm preferring it be more accurate than faster. I think that this way it's harder for the count to be wrong.